libs/blocks/ost/ost.js

```diff
@@ -1,15 +1,18 @@
-import { loadScript, loadStyle } from '../../utils/utils.js';
+import { getConfig, getLocale, loadScript, loadStyle } from '../../utils/utils.js';
+import { ENV_PROD, getTacocatEnv, getTacocatMetadata } from '../merch/utils.js';
 
 const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
 const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
 const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.js';
 const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.css';
 
-const ENVIRONMENT = 'PROD';
 const WCS_API_KEY = 'wcms-commerce-ims-ro-user-cc';
 const AOS_API_KEY = 'wcms-commerce-ims-user-prod';
 const CHECKOUT_CLIENT_ID = 'creative';
 
+window.adobeid = window.adobeid ?? {};
+window.adobeid.redirect_uri = 'https://mwpw-127984--milo--vladen.hlx.page/tools/ost';
```

Comment:
Is this uri right?

Reply:
No, this was an attempt to test it somehow, removed.